packages/eslint-config-react-app/index.js

```diff
@@ -241,15 +224,15 @@ module.exports = {
     ],
 
     // https://github.com/benmosher/eslint-plugin-import/tree/master/docs/rules
-    'import/first': 'error',
-    'import/no-amd': 'error',
-    'import/no-webpack-loader-syntax': 'error',
+    'import/first': 'warn',
+    'import/no-amd': 'warn',
+    'import/no-webpack-loader-syntax': 'warn',
 
     // https://github.com/yannickcr/eslint-plugin-react/tree/master/docs/rules
     'react/jsx-no-comment-textnodes': 'warn',
     'react/jsx-no-duplicate-props': ['warn', { ignoreCase: true }],
     'react/jsx-no-target-blank': 'warn',
-    'react/jsx-no-undef': 'error',
+    'react/jsx-no-undef': 'warn',
```

Comment:
This will always be a runtime error. Guaranteed. So there's no need to wait until the code executes. Please keep it an error.